content/en/security/application_security/guide/manage_account_theft_appsec.md

```diff
@@ -0,0 +1,669 @@
+
+To enable ASM using Remote Configuration, do the following:
+
+1. Go to [Remote Configuration][5].  
+2. Click **Get Started with ASM**.   
+3. In **Threat Management**, click **Select Services.**   
+4. Select your service(s), and then click **Next** and proceed with the setup instructions.
+
+When you see traces from your service in [ASM Traces][6], move to [Step 1.3: Validating login information is automatically collected](#step-1.3:-validating-login-information-is-automatically-collected).
+
+For more detailed instructions on using a new deployment, see [Enabling ASM Threat Detection using Datadog Tracing Libraries][7].
+
+### Step 1.3: Validating login information is automatically collected {#step-1.3:-validating-login-information-is-automatically-collected}
+
+After you have enabled ASM, you can validate that login information is collected by Datadog.
+
+**Note:** Once ASM is enabled on a service, wait a few minutes for users to log into the service or log into the service yourself.
+
+To validate login information is collected, do the following:
+
+1. Go to [Traces][8] in ASM.   
+2. Look for traces tagged with login activity from your login service. For example, in **Search for**, you might have `@appsec.security\_activity:business\_logic.users.login.\*`.  
+3. Check if all your login services are reporting login activity. You can see this in the **Service** facet.
+
+<!-- ![][image4] -->
+
+**If you don't see login activity from a service**, go to [Step 1.5: Manually instrumenting your services](#step-1.5:-manually-instrumenting-your-services).
+
+### Step 1.4: Validating login metadata is automatically collected {#step-1.4:-validating-login-metadata-is-automatically-collected}
+
+To validate that login metadata is collected, do the following:
+
+1. Go to [Traces][8] in ASM.   
+2. Look for traces tagged with successful and failed login activity from your login service. For example, in **Search for**, you might have all.  
+3. Open a trace.  
+4. In the trace details, is the **Security** tab, review **Business Logic Event**.  
+5. Check if the event has a false user.
+
+<!-- ![][image5] -->
+
+In the event of a **false** user (`usr.exists:false`), look for the following issues:
+
+- A single event: if the trace contains multiple login events, such as both successes and failures, go to [Step 1.5: Manually instrumenting your services](#step-1.5:-manually-instrumenting-your-services).  
+- If the event does not contain the mandatory metadata, it might appear as a user attribution section. The mandatory metadata is `usr.login` and `usr.exists` in the case of login failure, and `usr.id` in the case of login success. In this case, go to [Step 1.5: Manually instrumenting your services](#step-1.5:-manually-instrumenting-your-services).
```

Comment:
```suggestion
- Does the event does not contain the mandatory metadata? Some of it might appear as a user attribution section in case of login success. The mandatory metadata is `usr.login` and `usr.exists` in the case of login failure, `usr.login` and `usr.id` in the case of login success. In case some metadata is missing, go to [Step 1.5: Manually instrumenting your services](#step-1.5:-manually-instrumenting-your-services).
```